src/pretix/static/billing/js/event.js

```diff
@@ -0,0 +1,7 @@
+document.getElementById("back-btn").addEventListener("click", function () {
+    const basePath = JSON.parse(document.getElementById('base_path').textContent);
+    const url = window.location.href
+    const organizerMatch = url.match(/organizer\/([^/]+)/);
+    const organizerSlug = organizerMatch ? organizerMatch[1] : null;
+    window.location.href = `${basePath}/control/organizer/${organizerSlug}/settings/billing`;
```

Comment:
Hhmm, it is easy to see that the URL can be generated as `.../control/organizer/null/settings/billing`. Do you really want to make browser redirect to invalid URL (with `null`)?